### DISCUSSION TITLE
Switch is not rendered natively

### DISCUSSION THREAD

**Discussion by varand-pez** [tamagui/tamagui#1130] [Ideas]
### Current Behavior

Using the code sample in the [documentation](https://tamagui.dev/docs/components/switch/1.0.0), does not produce native Switch, comparing with the Switch component from core react native. Nor does it have animation.
I'm using iOS simulator with iOS 16.4.

### Expected Behavior

Expected behavior is to produce native Switch with correct animation.

### Tamagui Version

```markdown
"tamagui": "1.23.0",
```

### Reproduction

```markdown
My Code

...
<Label htmlFor="mySwitch">My Switch</Label>
<Switch
  id="mySwitch"
  size="$3"
  checked={checked}
  onCheckedChange={setChecked}
>
  <Switch.Thumb
    animation="bouncy"
    backgroundColor={
      checked ? Colors.primary : Colors.secondary
    }
  />
</Switch>
```

**Comment by natew** [comment]
Moved to discussion its a feature request

**Reply by varand-pez** [reply]
Hi Nate, thanks for your change.
Are you suggesting that rendering Switch natively is a feature request?
My confusions comes from documentation with native switch and the UX.

<img width="795" alt="Screenshot 2023-05-07 at 10 06 48 PM" src="https://user-images.githubusercontent.com/47469980/236738947-5e9d3620-81a5-4a2a-a05c-a01c86eb07b3.png">
